src/components/common/header/HelpNav.tsx

```diff
@@ -26,30 +28,52 @@ function HelpNav({
     const isEnterprise = currentServerInfo?.serverInfo?.installationType === InstallationType.ENTERPRISE
     const FEEDBACK_FORM_ID = `UheGN3KJ#source=${window.location.hostname}`
 
-    const HelpOptions: HelpOptionType[] = [
+    const CommonHelpOptions: HelpOptionType[] = [
         {
             name: 'View documentation',
             link: DOCUMENTATION.HOME_PAGE,
             icon: File,
             showSeparator: true,
         },
+        
         {
-            name: 'Chat with support',
+            name: 'Join discord community',
             link: 'https://discord.devtron.ai/',
-            icon: Chat,
-            showSeparator: !isEnterprise,
+            icon: Discord,
+            showSeparator: isEnterprise,
+        },
+        
+    ]
+
+    const EnterpriseHelpOptions: HelpOptionType[] = [
```

Comment:
move all this const to const files

Reply:
Created a new constant file and moved these constants to it.